internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientTest.java

```diff
@@ -204,6 +204,13 @@ public void testBatchStore(DaVinciConfig clientConfig) throws Exception {
     MetricsRepository metricsRepository = new MetricsRepository();
 
     // Test record transformation
+
+    // Turn on GZIP compression
+    cluster.useControllerClient(controllerClient -> {
```

Comment:
So this is gonna get noOp compression because compression will only get enabled on the next version of the store. 

Follow this call up with a cluster.getNewVersion() call.  This will force the store to take the new copmression config.